README.md

```diff
@@ -53,7 +53,7 @@ Install tools and configuration manually:
    * Install Visual C++ Build Environment: [Visual Studio Build Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools)
    (using "Visual C++ build tools" workload) or [Visual Studio Community](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community)
    (using the "Desktop development with C++" workload)
-   * Launch cmd, `npm config set msvs_version 2017`
```

Comment:
This is actually optional, I am inclined to delete this setup. @cclauss @rvagg Second opinion ?

Reply:
I am not a Windows user.

Reply:
@beliaev-maksim Can you delete this line ? I think for most user, this is not necessary.

Reply:
@gengjiawen @cclauss
Maybe we completely remove the part with visual studio?
To my knowledge new versions of Node come directly with additional developer tools (although, you have to check it while install it)

That should provide better integration
What do you think?

Reply:
> Maybe we completely remove the part with visual studio?

Not a good idea, many people use visual studio directly. They have theire own cpp config.